systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -380,13 +380,15 @@ private void testDockerImagesForKafkaCluster(String clusterName, int kafkaPods,
 
         //Verifying docker image for zookeeper pods
         for (int i = 0; i < zkPods; i++) {
-            String imgFromPod = getImageNameFromPod(zookeeperPodName(clusterName, i));
+            String imgFromPod = getContainerImageNameFromPod(zookeeperPodName(clusterName, i), zookeeperClusterName(clusterName));
```

Comment:
I think that now when we have more containers, we should not name the main container based on the cluster name. It should be named based on its function ... so here I would expect the name to be for example `zookeeper`.

Reply:
Should we do the same for KafkaCluster, KafkaConnect, TopicOperator ? Avoiding mixing things I would open a new PR for changing on both.

Reply:
Because you need it here and not yet there. You can change it there when you add the TLS sidecar to Kafka.

Reply:
well from the next for loop we are getting container name for Kafka as well (and for TO at the end of the method).